app/views/includes/scripts.html

```diff
@@ -3,6 +3,7 @@
   <script src="{{scriptUrl}}"></script>
 {% endfor %}
 
+<script src="/public/javascripts/kit.js"></script>
```

Comment:
I'm wondering about consistency of approach - in application.scss we write to one file - out code and the users code. Is that the right approach to use here too?

Reply:
I thought about this, but I don't think there is an equivalent for the javascript, we don't compile our JS in any way, we just copy it to the public folder.

I'm not sure that there's any downside to the inconsistency though, can you think of any?

Reply:
good points thanks, I just thought if we could be consistent its an easier mental model for us to maintain the codebase